test/reporter_output_test.cc

```diff
@@ -27,9 +27,9 @@ BENCHMARK(BM_basic);
 
 ADD_CASES(TC_ConsoleOut, {{"^BM_basic %console_report$"}});
 ADD_CASES(TC_JSONOut, {{"\"name\": \"BM_basic\",$"},
-                       {"\"iterations\": %int,$", MR_Next},
-                       {"\"real_time\": %int,$", MR_Next},
-                       {"\"cpu_time\": %int,$", MR_Next},
+                       {"\"iterations\": %float,$", MR_Next},
```

Comment:
Hmm, you're right, this is not right, likely too greedy s///
https://github.com/google/benchmark/pull/431